contracts/src/_testing/integration/IntegrationTestTrueVerifier.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: AGPL-3.0
-pragma solidity 0.8.26;
+pragma solidity ^0.8.26;
```

Comment:
this is a top level contract and should be without the ^ - understandably it is a test one, but for consistency sake it should change

Reply:
fixed